pnpm-lock.yaml

```diff
@@ -5139,6 +5139,7 @@ packages:
     engines: {node: '>=8'}
     os: [darwin]
     hasBin: true
+    requiresBuild: true
```

Comment:
Can you please revert this? It doesn't seem relevant to the changes you're looking to add